components/SignUpForm/SignUpForm.tsx

```diff
@@ -45,46 +47,41 @@ export const SignUpForm = () => {
     createUserWithEmailAndPassword(auth, data.email, data.password)
       .then(async (userCredential) => {
         const user = userCredential.user;
-        await router.push(routes.SIGNIN);
+        await router.push(routes.HOME);
```

Comment:
routes? I should make a choice, or you keep all routes in routes variable or you write row inline